.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - ci
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - plugin: "livesync"
+          - plugin: "payment_paypal"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        id: cache-pip
+        with:
+          path: .venv
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements*.txt') }}
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        if: steps.cache-pip.outputs.cache-hit != 'true'
+        with:
+          python-version: 3.9
+      - name: Get branch name (base)
+        if: github.event_name != 'pull_request'
+        shell: bash
+        run: echo "BRANCH_NAME=$(echo ${GITHUB_REF} | sed 's/.*\///')" >> $GITHUB_ENV
+      - name: Get branch name (pull request)
+        if: github.event_name == 'pull_request'
+        shell: bash
+        run: echo "BRANCH_NAME=$(echo ${GITHUB_HEAD_REF} | sed 's/.*\///')" >> $GITHUB_ENV
+      - name: Install python dependencies
+        if: steps.cache-pip.outputs.cache-hit != 'true'
+        run: |
+          python3 -m venv .venv
+          source .venv/bin/activate
+          pip install -U pip setuptools
+          pip install wheel
+          pip install -e git+https://github.com/indico/indico.git@${BRANCH_NAME}#egg=indico
+      - name: Activate virtualenv for later steps
+        run: |
+          echo "VIRTUAL_ENV=$(pwd)/.venv" >> $GITHUB_ENV
+          echo "$(pwd)/.venv/bin" >> $GITHUB_PATH
+      - name: Install
+        env:
+          PLUGIN: ${{ matrix.plugin }}
```

Comment:
Can't you use `${{ matrix.plugin }}` directly in the `run` command below?

Reply:
I tested it on a separate branch and couldn't do it myself. I'm not sure if this is possible.
https://github.com/plourenco/indico-plugins/runs/1530852142?check_suite_focus=true (expand line 1)

Reply:
I couldn't do it previously, but the mistake was mine. I'm going to try that again.